source/installation/install-on-a-single-machine/install-rhel.txt

```diff
@@ -1,5 +1,75 @@
+.. _single-rhel:
+
 ============
 Install RHEL
 ============
 
-TBD.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator on a single RHEL/CentOS machine. This 
+installation method uses default settings and binds Relational Migrator to a single 
+IP and port.
+
+Steps
+-----
+
```

Comment:
Sorry about this @jocelyn-mendez1! Ian and I talked about the procedure directive. I think we'll want to aim for adding the procedure directive later. But most of these tasks were not written with a real procedure in mind and their formatting and wording is a little off. Would you mind switching to an ordered list, for now?

Reply:
Done.